docs/source/development-guidelines.rst

```diff
@@ -0,0 +1,221 @@
+Zipline Development Guidelines
+==============================
+This page is intended for developers of Zipline, people who want to contribute to the Zipline codebase or documentation, or people who want to install from source and make local changes to their copy of Zipline.
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements and ideas are welcome. We `track issues`__ on `GitHub`__ and also have a `mailing list`__ where you can ask questions.
+
+__ https://github.com/quantopian/zipline/issues
+__ https://github.com/
+__ https://groups.google.com/forum/#!forum/zipline
+
+Creating a Development Environment
+----------------------------------
+
+First, you'll need to clone Zipline by running:
+
+.. code-block:: bash
+
+   git clone git@github.com:your-github-username/zipline.git
+
+Then check out to a new branch where you can make your changes:
+
+.. code-block:: bash
+		
+   git checkout -b some-short-descriptive-name
+
+If you don't already have them, you'll need some C library dependencies. You can follow the `install guide`__ to get the appropriate dependencies.
+
+__ http://www.zipline.io/install.html 
+
+The following section assumes you already have virtualenvwrapper and pip installed on your system. Suggested installation of Python library dependencies used for development:
+
+.. code-block:: bash
+
+   $ mkvirtualenv zipline
+   $ ./etc/ordered_pip.sh ./etc/requirements.txt
+   $ pip install -r ./etc/requirements_dev.txt
+   $ pip install -r ./etc/requirements_blaze.txt 
+   $ pip install -r ./etc/requirements_talib.txt 
+
+Finally, you can build the C extensions by running:
+
+.. code-block:: bash
+
+   $ python setup.py build_ext --inplace
+
+To finish, make sure tests pass by running
+
+.. code-block:: bash
+
+   $ nosetests
```

Comment:
Will this fail because we haven't gotten to the "Running Tests" section yet, and the user hasn't installed TA-lib?